### DISCUSSION TITLE
List of WL products necessary to read for cosmological inference

### DISCUSSION THREAD

**Discussion by gcanasherrera** [euclidlib/euclidlib#38] [General]
List of products/information that we need to retrieve from the Euclid SGS and SWGs to perform the Euclid cosmological inference exercise for Cosmic Shear and 3x2pt. If clicked, it means that the reading function is in euclidlib either in main or in a branch.

Link to the Euclid DPDD: http://st-dm.pages.euclid-sgs.uk/data-product-doc/dm10/index.html

- [x]  C_ells (already included in `euclidlib`)
- [x]  COSEBIs (already included in `euclidlib`)
- [x]  mixing matrices (already included in `euclidlib`)
- [x] 2pcf (already included in `euclidlib`)
- [x] n(z)
- [x] band powers
- [ ]  CM-C_ells
- [ ]  CM-2pcf
- [ ]  prior information for multiplicative bias after calibration
- [ ]  prior information about deltaZ (per-bin redshift bins)

**Comment by gcanasherrera** [comment]
@annmapo to provide feedback about possible missing products interesting for the cosmological inference analysis (thanks!)

**Reply by annmapo** [reply]
For the lenses, we will also need prior information about the stretch of each n(z) (change in the width, per bin).  
We will also need an estimate of the lens magnification in each redshift bin (with an associated error, if possible). 
Besides this, some estimate of the galaxy bias values will be important for the covariance matrix (it impacts the error bars). 

The covariance matrix for COSEBIs is also needed, right?
